crates/ruma-client-api/src/lib.rs

```diff
@@ -15,6 +15,7 @@ pub mod appservice;
 pub mod backup;
 pub mod config;
 pub mod context;
+pub mod dehydrated_device;
```

Comment:
This module should be behind an unstable cargo feature.

Reply:
Feature added in c6cc430f29d2b468a71590c6f4446fe6afccef48. I have forgotten if I need to do anything with xtask when we add new features, do I need to?

Reply:
You also need to add the cargo feature in the `ruma` crate, and add it to the `__ci` feature there too.

Reply:
Ah duh, thanks for reminding me, fixed in https://github.com/ruma/ruma/pull/1605/commits/b64f87cc47d709867f08bb6ad262ca5fb8a3026d.